browser/ui/brave_layout_constants.cc

```diff
@@ -21,9 +21,9 @@ absl::optional<int> GetBraveLayoutConstant(LayoutConstant constant) {
   //     ui::MaterialDesignController::IsNewerMaterialUi();
   switch (constant) {
     case TAB_HEIGHT: {
-      const int tab_height = HorizontalTabsUpdateEnabled() ? 36 : 30;
-      return (touch ? 41 : tab_height) +
-             GetLayoutConstant(TABSTRIP_TOOLBAR_OVERLAP);
+      const int tab_height =
+          HorizontalTabsUpdateEnabled() ? 44 : (touch ? 41 : 30);
```

Comment:
Do you think it would be appropriate to give these values descriptive constant names?

Reply:
The 44 value, I think so. The other values (existing code used when the flag is off) I'll leave as is.